src/Jackett.Common/Indexers/FunFile.cs

```diff
@@ -77,27 +77,17 @@ protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuer
         {
             var releases = new List<ReleaseInfo>();
 
-            var searchString = query.GetQueryString();
-            var searchUrl = SearchUrl;
-            var queryCollection = new NameValueCollection();
-            queryCollection.Add("incldead", "1");
-            queryCollection.Add("showspam", "1");
-
-            if (!string.IsNullOrWhiteSpace(searchString))
-            {
-                queryCollection.Add("search", searchString);
-            }
-
             var cats = MapTorznabCapsToTrackers(query);
```

Comment:
If we use `.FirstOrDefault()` we can avoid creating a variable here.
```suggestion

```

Reply:
done